apps/modernization-api/src/main/resources/graphql/patient-search.graphqls

```diff
@@ -84,14 +91,14 @@ type PatientSearchResult {
   addresses: [PatientSearchResultAddress!]!
   phones: [String!]!
   emails: [String!]!
+  detailedPhones: [PatientSearchResultPhone!]!
```

Comment:
More of a question than a review, but does this new `detailedPhones `overlap with the same data in `phones`, and if so, will we still need it?

Reply:
Yes `phones` would be replaced by `detailedPhones`.  Unfortunately phones is currently an array of strings so I couldn't just supplement it with the added fields and replacing it would break the front-end code.